flang/test/Lower/OpenMP/infinite-loop-in-construct.f90

```diff
@@ -0,0 +1,19 @@
+! RUN: bbc -fopenmp -o - %s | FileCheck %s
+
+! Check that this test can be lowered successfully.
+! See https://github.com/llvm/llvm-project/issues/74348
+
+! CHECK-LABEL: func.func @_QPsb
+! CHECK: omp.parallel
```

Comment:
I think it would be a bit better to also check the general structure inside of omp.parallel. That is, checking the blocks and terminators that are created.

Reply:
That's done.